src/test/java/api/loans/ChangeDueDateAPITests.java

```diff
@@ -0,0 +1,285 @@
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+
+    checkInFixture.checkInByBarcode(item);
+
+    final Response response = changeDueDateFixture
+      .attemptChangeDueDate(new ChangeDueDateRequestBuilder()
+        .forLoan(loan.getId())
+        .withDueDate(newDueDate));
+
+    assertResponseOf(response, 422, "id", loan.getId());
+    assertResponseMessage(response, "Loan is closed");
+  }
+
+  @Test
+  public void cannotChangeDueDateWhenDeclaredLost() {
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+
+    assertThat(loansFixture.declareItemLost(loan.getJson()),
+      hasStatus(HTTP_NO_CONTENT));
+
+    final Response response = changeDueDateFixture
+      .attemptChangeDueDate(new ChangeDueDateRequestBuilder()
+        .forLoan(loan.getId())
+        .withDueDate(newDueDate));
+
+    assertResponseOf(response, 422, ITEM_ID, item.getId());
+    assertResponseMessage(response, "item is Declared lost");
+  }
+
+  @Test
+  public void cannotChangeDueDateWhenClaimedReturned() {
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+
+    assertThat(loansFixture.claimItemReturned(
+      new ClaimItemReturnedRequestBuilder()
+        .forLoan(loan.getId().toString())
+       ), hasStatus(HTTP_NO_CONTENT));
+
+    (new ChangeDueDateRequestBuilder()).forLoan(loan.getId().toString());
+
+    final Response response = changeDueDateFixture
+      .attemptChangeDueDate(new ChangeDueDateRequestBuilder()
+        .forLoan(loan.getId())
+        .withDueDate(newDueDate));
+
+    assertResponseOf(response, 422, ITEM_ID, item.getId());
+    assertResponseMessage(response, "item is Claimed returned");
+  }
+
+  @Test
+  public void cannotReapplyDueDateWhenClaimedReturned() {
+    assertThat(loansFixture.claimItemReturned(
+      new ClaimItemReturnedRequestBuilder()
+        .forLoan(loan.getId().toString())
+      ), hasStatus(HTTP_NO_CONTENT));
+
+    final Response response = changeDueDateFixture
+      .attemptChangeDueDate(new ChangeDueDateRequestBuilder()
+        .forLoan(loan.getId())
+        .withDueDate(dueDate));
+
+    assertResponseOf(response, 422, ITEM_ID, item.getId());
+    assertResponseMessage(response, "item is Claimed returned");
+  }
+
+  @Test
+  public void canChangeDueDateWithOpenRequest() {
+    final DateTime newDueDate = dueDate.plus(Period.days(14));
+
+    requestsFixture.place(new RequestBuilder()
+      .hold()
+      .forItem(item)
+      .by(usersFixture.steve())
+      .fulfilToHoldShelf(servicePointsFixture.cd1()));
+
+    changeDueDateFixture.changeDueDate(new ChangeDueDateRequestBuilder()
+      .forLoan(loan.getId())
+      .withDueDate(newDueDate));
+
+    Response response = loansClient.getById(loan.getId());
+
+    JsonObject updatedLoan = response.getJson();
+
+    assertThat("due date is not updated",
+      updatedLoan.getString("dueDate"), isEquivalentTo(newDueDate));
+  }
+
+  @Test
+  public void changeDueDateNoticeIsSentWhenPolicyIsDefined() {
```

Comment:
Does there need to be a test for not sending a notice when there isn't a policy in place?

Reply:
That is seems like something that needs to be tested.
It would be good to make sure that a policy is not sent in that case.